dashboard/app/controllers/registrations_controller.rb

```diff
@@ -54,6 +54,14 @@ def sign_up_params
         params[:email_preference_source] = EmailPreference::ACCOUNT_SIGN_UP
         params[:email_preference_form_kind] = "0"
       end
+
+      if params[:data_transfer_agreement_required] && params[:data_transfer_agreement] == "1"
+        params[:data_transfer_agreement_accepted] = true
+        params[:data_transfer_agreement_request_ip] = request.ip
+        params[:data_transfer_agreement_source] = User::ACCOUNT_SIGN_UP
+        params[:data_transfer_agreement_kind] = "0"
```

Comment:
Just confirming we want a string here?  I have a number; I'll update.

Reply:
I'm mirroring what we do for `EmailPreference` (see https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/email_preference.rb#L10) which is a string, so let's do a string here too.